src/UhdmAst.cpp

```diff
@@ -983,6 +1009,15 @@ namespace UhdmAst {
         return new AstDot(new FileLine("uhdm"), false, lhsNode, rhsNode);
       }
       case vpiRefObj: {
+        // Recover the original package, if any
+        std::string is_imported;
+        if (auto s = vpi_get_str(vpiImported, obj_h)) {
+          is_imported = s;
+        }
+        if (is_imported != "") {
+          objectName = is_imported +"::"+ objectName;
+        }
```

Comment:
Do you really need the `is_imported` variable here, two ifs and the copy of the string? Cannot this be merged into sth like:
```
        if (auto pkg_name = vpi_get_str(vpiImported, obj_h)) {
          objectName = pkg_name +"::"+ objectName;
        }
```

Reply:
@rkapuscik I am not sure what happened here. In the current version you completely removed the code that extends the object name? Was that on purpose?

Reply:
Yes, when refactoring `is_imported` I noticed that the only cases when this was needed were eliminated with https://github.com/antmicro/verilator/pull/23/commits/b0ed3dbebcd456dc1abf841138f5d8ac6e0602d5, so this code was redundant.